_pages/home/spartacus-faq.md

```diff
@@ -3,131 +3,140 @@ title: Spartacus FAQ
 
-### How long does it take to get a Spartacus-based storefront up and running? 
+### How long does it take to get a Spartacus-based storefront up and running?
 
 Assuming a vanilla SAP Commerce Cloud B2C backend is up and running and configured to accept OCC API calls, a developer can get create a Spartacus-based storefront within 10 minutes. See [Building the Spartacus Storefront from Libraries]({{ site.baseurl }}{% link _pages/install/building-the-spartacus-storefront-from-libraries.md %}).
 
-### Is there a demo of a Spartacus-based storefront available? 
+### Is there a demo of a Spartacus-based storefront available?
 
 Try out our continuous-integration server [here](https://sap.github.io/cloud-commerce-spartacus-storefront). As of this writing (April 19, 2019), note that the backend server used by this sample site is not up to date with latest 1905, due to the imminent release of 1905 late May. The server will be updated shortly after release.  
 
-### What enablement options exist for helping me understand how to use Spartacus? Is training available? 
+### What enablement options exist for helping me understand how to use Spartacus? Is training available?
 
 Documentation is available from the Spartacus GitHub Pages [documentation site](https://sap.github.io/cloud-commerce-spartacus-storefront-docs/). We are starting a collection of [helpful how-to videos](https://enable.cx.sap.com/tag/tagid/spartacus). Official training via SAP Education is not planned for launch but may be made available in the future.  
 
-### Can I customize Spartacus? 
+### Can I customize Spartacus?
 
 Spartacus is a set of libraries that contain core libraries, components and styling. You can choose which versions of the libraries you would like to use in your application, and these can be used to fully configure and customize your storefront. However, customization is not done in the same way as with Accelerator; you never customize Spartacus code directly – rather, you overwride or replace styling and code. This approach allows ease of upgradability. 
 
-### What is the release cycle for Spartacus? How do I update? 
+### What is the release cycle for Spartacus? How do I update?
 
 Spartacus releases are independent of the release cycle of SAP Commerce Cloud. Generally, new Spartacus libraries will be released every 2 weeks. For the next six months at least, changes will include new features or fixes only, with no changes that affect compatibility with existing applications. (Spartacus follows semantic versioning 2.0.) On the development side, you are in total control of whether to accept library updates. Note that some features in Spartacus may be delayed or not available if updates to platform OCC APIs are required. A new major release (1.2.3 > 2.0.0 for example, where changes affect ease of upgradability) will be announced ahead of time and will likely not happen until end of 2019 or early 2020. 
```

Comment:
outdated text "for the next six months"

Reply:
"How do I update?" removed from title, and everything after the first sentence has been removed.